### PR TITLE
adding colors brown, black and white

### PR DESCRIPTION
**Pull request by cdesch**
I've added:

-  black and white to `lib/faker/color/en.ex`
-  brown to `lib/faker/commerce/en.ex`

### CODE REVIEW

**Comment by cdesch** [comment]
Also adding `Industry` to options based off of the [Industry Classification Benchmark
](https://en.wikipedia.org/wiki/Industry_Classification_Benchmark)

**Comment by cdesch** [comment]
I fixed the code to pass the credo tests.

File: lib/faker/industry/en.ex [hunk 1]
```diff
@@ -0,0 +1,32 @@
+defmodule Faker.Industry.En do
+  import Faker, only: [sampler: 2]
+
+  @moduledoc """
+  Functions for generating Beer related data in English
```
Comment (igas) [inline lib/faker/industry/en.ex]: Old description

File: lib/faker/color/en.ex [hunk 2]
```diff
@@ -9,7 +9,7 @@ defmodule Faker.Color.En do
   Returns a random english color name
   """
   @spec name() :: String.t
-  sampler :name, ["Red", "Green", "Blue", "Pink", "Brown", "Orange", "Yellow", "Purple"]
+  sampler :name, ["Red", "Green", "Black", "Blue", "Pink", "Brown", "Orange", "Yellow", "Purple", "White"]
```
Comment (igas) [inline lib/faker/color/en.ex]: Good catch

**Comment by igas** [comment]
Please add yourself to changelog if you want.

File: lib/faker/industry/en.ex (hunk 1 above)
Comment (cdesch) [reply lib/faker/industry/en.ex]: Fixed in the latest PR.

**Comment by cdesch** [comment]
I fixed the description name in the latest set of commits, added docs for Industry Segments in [industry.md](industry.md), and update the changelog with the and [usage.md](usage.md) to reflect the changes.

**Comment by igas** [comment]
Thank you for your contribution! Please send separate PRs next time for bugfix and for new features.
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Comment by cdesch** [comment]
@igas Will do. Thank you

**Pull request merged**